MQRS.md

```diff
@@ -1,11 +1,13 @@
 # Overview
-Tables: MORT_ELIG_CRITERIA, SECTOR_YEAR_ROSTER
+Tables: MORT_ELIG_CRITERIA,
 
-Location: Sole
+Location: NEFSCDB1
 
-Schema: MQRS
+Schema: NEFSC_GARFO.MQRS_
 
 The MQRS system is fun. It contains information on "eligibilities."  The sector rosters are also stored here.  It is used in conjunction with other tables, but not cross-referenced against them for logical consistency.
+
+The SECTOR schema still exists on SOLE.
```

Comment:
Sole is going away so what does this mean?

Reply:
Sole is still there (as of today), so I don't want to delete this line right now.

Reply:
issue #31. We'll pull it when sole is turned off.